urbanairship-react-native/js/UAEventEmitter.ts

```diff
@@ -1,33 +1,38 @@
 /* Copyright Airship and Contributors */
```

Comment:
Any reason this class isn't renamed "EventEmitter", like the other classes?

Reply:
There's already an EventEmitter in the react native typings, and this one derives from it. I thought about changing the name to something else, but I couldn't come up with something more descriptive so I left it alone.